tests/consumer.lua

```diff
@@ -100,6 +100,8 @@ local function consume(timeout)
                 if err ~= nil then
                     log.error("got error '%s' while commiting msg from topic '%s'", err, msg:topic())
                 end
+            else
```

Comment:
I don't see "producer:destroy()" and "consumer:destroy()" calls in tests. How did you check it works?

Reply:
My bad. It was done in init.lua